Geometry/HGCalCommonData/interface/HGCalWafer.h

```diff
@@ -0,0 +1,50 @@
+#ifndef Geometry_HGCalCommonData_HGCalWafer_h
+#define Geometry_HGCalCommonData_HGCalWafer_h
+
+#include <cmath>
+#include <cstdint>
+
+class HGCalWafer {
+public:
+  HGCalWafer(double waferSize, int32_t nFine, int32_t nCoarse);
+
+  static constexpr int32_t WaferOrient0 = 0;
+  static constexpr int32_t WaferOrient1 = 1;
+  static constexpr int32_t WaferOrient2 = 2;
+  static constexpr int32_t WaferOrient3 = 3;
+  static constexpr int32_t WaferOrient4 = 4;
+  static constexpr int32_t WaferOrient5 = 5;
+
+  static constexpr int32_t WaferPlacementIndex0 = 0;
+  static constexpr int32_t WaferPlacementIndex1 = 1;
+  static constexpr int32_t WaferPlacementIndex2 = 2;
+  static constexpr int32_t WaferPlacementIndex3 = 3;
+  static constexpr int32_t WaferPlacementIndex4 = 4;
+  static constexpr int32_t WaferPlacementIndex5 = 5;
+  static constexpr int32_t WaferPlacementIndex6 = 6;
+  static constexpr int32_t WaferPlacementIndex7 = 7;
+  static constexpr int32_t WaferPlacementIndex8 = 8;
+  static constexpr int32_t WaferPlacementIndex9 = 9;
+  static constexpr int32_t WaferPlacementIndex10 = 10;
+  static constexpr int32_t WaferPlacementIndex11 = 11;
+
+  static constexpr int32_t WaferPlacementExtra = 6;
+  static constexpr int32_t WaferPlacementOld = 7;
+  static constexpr int32_t WaferPlacementTotal = 12;
+
+  static constexpr int32_t CornerCell = 0;
+  static constexpr int32_t TruncatedCell = 1;
+  static constexpr int32_t ExtendedCell = 2;
+
+  std::pair<double, double> HGCalWaferUV2XY1(int32_t u, int32_t v, int32_t placementIndex, int32_t type);
+  std::pair<double, double> HGCalWaferUV2XY2(int32_t u, int32_t v, int32_t placementIndex, int32_t type);
+  std::pair<int32_t, int32_t> HGCalWaferUV2Cell(int32_t u, int32_t v, int32_t placementIndex, int32_t type);
+  static int32_t HGCalWaferPlacementIndex(int32_t iz, int32_t fwdBack, int32_t orient);
+
+private:
+  const double sqrt3By2_ = (0.5 * std::sqrt(3.0));
+  int32_t N_[2];
+  double R_[2], r_[2];
```

Comment:
These variable names go against the CMS naming [guidelines](http://cms-sw.github.io/cms_coding_rules.html):

2.12 Do not use single character names, except for loop indices.
2.16 Use clear and explanatory variable names.

Please change them to help the reader understand them and to make them easier to search for in the code. Also, the names should be camel-case, starting with lower case.

Reply:
Changed names of variables